src/app/donation-start/donation-start-form/donation-start-form.component.html

```diff
@@ -493,6 +493,10 @@ <h3 class="b-rt-0 b-m-0 b-bold"><span class="span-hr"></span>How does Big Give u
                     Please note that you might continue to receive communications from the charity if you have already shared your details with them via other methods.
                   </p>
                 }
+                @if (marketingGroup.value.optInCharityEmail === true) {
```

Comment:
I guess we could simplify like this?
```suggestion
                @if (marketingGroup.value.optInCharityEmail) {
```